plugins/chunter-resources/src/backlinks.ts

```diff
@@ -23,10 +26,20 @@ export function getTime (time: number): string {
 export function isToday (time: number): boolean {
   const current = new Date()
   const target = new Date(time)
-  return current.getDate() === target.getDate() && current.getMonth() === target.getMonth() && current.getFullYear() === target.getFullYear()
+  return (
+    current.getDate() === target.getDate() &&
+    current.getMonth() === target.getMonth() &&
+    current.getFullYear() === target.getFullYear()
+  )
 }
 
-function extractBacklinks (backlinkId: Ref<Doc>, backlinkClass: Ref<Class<Doc>>, attachedDocId: Ref<Doc> | undefined, message: string, kids: NodeListOf<ChildNode>): Array<Data<Backlink>> {
+function extractBacklinks (
+  backlinkId: Ref<Doc>,
+  backlinkClass: Ref<Class<Doc>>,
+  attachedDocId: Ref<Doc> | undefined,
+  message: string,
+  kids: NodeListOf<ChildNode>
+): Array<Data<Backlink>> {
```

Comment:
Why not ?
```suggestion
): Data<Backlink>[] {
```

Reply:
Either way `npm run format` converts this back to `Array<...>`